bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -148,12 +152,36 @@ <h4 class="heading-linux">{{_('Download Firefox <span>for Linux</span>') }}</h4>
     </section>
   </div>
 </div>
+
+{% if l10n_has_tag('fxnew_account_032019') %}
+<div id="firefox-account">
+  <h4 class="fxa-title">{{ _('You’ve already got the Firefox browser. Now get everything else Firefox.') }}</h4>
+  <p class="fxa-intro">{{ _('Sync bookmarks, send tabs, save to Pocket and more with a Firefox Account.') }}</p>
+
+  {{ fxa_email_form(entrypoint='firefox-new', button_text=_('Get a Firefox Account'), utm_source='firefox-new', utm_params={'campaign': 'firefox-new', 'content': 'firefox-new-fxa', 'term': 'existing-users', 'medium': 'referral'}) }}
+
+  <p class="fxa-signin">{{ _('Already have an account?') }} <a {{ fxa_link_fragment(entrypoint='accounts-page', utm_params={'campaign': 'fxa-embedded-form', 'content': '/firefox/accounts/', 'source': 'accounts-page'}) }}>{{ _('Sign In') }}</a></p>
+
+  {{ download_firefox(dom_id='download-firefox-account-modal', alt_copy=_('Download Anyway'), button_color='button-hollow button-blue', locale_in_transition=True, download_location='other') }}
+
+  <ul class="options">
+    <li><a href="{{ url('firefox.all') }}">{{ _('Download in another language') }}</a></li>
+    <li class="option-refresh"><a id="refresh-firefox" href="https://support.mozilla.org/kb/refresh-firefox-reset-add-ons-and-settings?utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=learn-more-link"data-interaction="Refresh Firefox" data-element-action="Firefox Desktop" data-button-name="Refresh Firefox" data-cta-position="Primary">{{ _('Fix a problem') }}</a>
```

Comment:
I made it a link since without JS it just goes directly to SUMO. But I agree, I think we need more context than just "fix a problem". Maybe we still need a button + link treatment and we can improve the labeling.

Reply:
Perhaps a `role="button"` would suffice if JS is enabled?

Reply:
It's just a plain link now, and outside the modal.